package.json

```diff
@@ -10,6 +10,9 @@
     "type": "git",
     "url": "adobe/asset-compute-sdk"
   },
+  "engines": {
+    "node": ">=12.0.0"
+  },
```

Comment:
this ensures it can only be installed with node >=12

Reply:
And we need this because there is a breakage on node 10? Or just because we don't support node 10 anymore?

Reply:
I think because there is a breakage on node 10? with the dips updates.. , is that so @jdelbick ?